server.go

```diff
@@ -813,6 +813,11 @@ func (l *listenSocket) Close() error {
 // Serve returns when lis.Accept fails with fatal errors.  lis will be closed when
 // this method returns.
 // Serve will return a non-nil error unless Stop or GracefulStop is called.
+//
+// Note: Go overrides OS defaults for TCP keepalive time and interval to 15s.
+// To retain OS defaults, pass a net.Listener created by calling the Listen method
+// on a net.ListenConfig with the `KeepAlive` field set to a negative value. See
+// helloworld/greeter_server/main.go for an example.
```

Comment:
Mention Go version and link to github issue as above, please.

Also, remove the reference to the example.  We can add an example for it later if needed, or we could post the code snippet in #6250 and link there.